README.md

```diff
@@ -213,6 +215,9 @@ clasp <command>
 
 #### Run Tests (experimental)
 
+Check out `/tests/test.ts` and change `describe.skip(...)` to `describe(...)` for relevant tests.
+If you created a new command, please make a test for it. Here's how to run the tests:
```

Comment:
Delete: `If you created a new command, please make a test for it.`

We can add this when tests are common and include it somewhere else.